Library/Homebrew/cask/installer.rb

```diff
@@ -124,6 +125,17 @@ def install
       raise
     end
 
+    def check_deprecate_disable
+      deprecate_disable_type = DeprecateDisable.type(@cask)
+      return if deprecate_disable_type.blank?
```

Comment:
```suggestion
      return if deprecate_disable_type.nil?
```

Nit: `DeprecateDisable.type` cannot return a blank string.